.github/dependabot.yml

```diff
@@ -12,9 +12,16 @@ updates:
     schedule:
       interval: "weekly"
     ignore:
+      # Managed by cisagov/skeleton-generic
       - dependency-name: actions/cache
       - dependency-name: actions/checkout
+      - dependency-name: actions/setup-go
       - dependency-name: actions/setup-python
+      - dependency-name: hashicorp/setup-terraform
+      - dependency-name: mxschmitt/action-tmate
+      # # Managed by cisagov/skeleton-python-library
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/upload-artifact
```

Comment:
```suggestion
      # Managed by cisagov/skeleton-python-library
      - dependency-name: actions/download-artifact
      - dependency-name: actions/upload-artifact
```

Reply:
FWIW, I have been doing a `git diff origin/develop` after resolving the conflicts (but before force-pushing) to look for these sorts of oversights.

Reply:
Thanks - I knew I was forgetting something last night. 👍